nuxeo-runtime/nuxeo-launcher-commons/src/test/java/org/nuxeo/launcher/config/ConfigurationGeneratorTest.java

```diff
@@ -514,8 +514,9 @@ public void testEnvironmentVariablesExpansion() throws Exception {
 
     @Test
     public void testEnvironmentVariableInTemplates() throws Exception {
-        configGenerator.getUserConfig().setProperty(ConfigurationGenerator.PARAM_TEMPLATES_NAME, "${env:NUXEO_DB_TYPE:default,sql},docker,${env:NUXEO_DB_HOST:docker}");
-        assertEquals("default,sql,docker,10.0.0.1",String.join(",",configGenerator.getTemplateList()));
+        configGenerator.getUserConfig().setProperty(ConfigurationGenerator.PARAM_TEMPLATES_NAME,
+                "${env:NUXEO_DB_TYPE:default},docker,${env:NUXEO_DB_HOST:docker}");
+        assertEquals("default,docker,10.0.0.1",String.join(",",configGenerator.getTemplateList()));
```

Comment:
There's spaces missing between the expected value and the actual one; between the 2 params of `String#join`.